centos-8-stream/create-image-overrides.sh

```diff
@@ -0,0 +1,37 @@
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2020 Joyent, Inc.
+#
+
+# Sets global variables iso_file and iso_sha
+get_iso_file_sha() {
+    local file
+
+    if [[ -n $iso_file ]]; then
+        return
+    fi
+
+    echo "Checking to see if we have the iso for $distro $release:"
+    for file in CHECKSUM CHECKSUM.asc; do
```

Comment:
I'm not sure how often this is the case,  but currently there is only CHECKSUM at https://mirrors.edge.kernel.org/centos/8-stream/isos/x86_64/